### PR TITLE
Fix loading of the rsstorrent cookies and url attributes.

### LINKED ISSUE #3130: Custom torrent providers information lost after restart
After restart **RSS URL** and **Cookies (optional)** are missing from the custom provider. Latest develop.

### PR DESCRIPTION
**Pull request by p0psicles**
* Got lost after restart.

- [x] PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [x] Read the [contribution guide](https://github.com/pymedusa/Medusa/blob/master/.github/CONTRIBUTING.md)

Should really fix #3130

### CODE REVIEW

**Comment by medariox** [comment]
Sadly still the same issue here.

**Comment by p0psicles** [comment]
Hmm strange, this definitely fixed it for me?

**Comment by medariox** [comment]
You are completely right, for some reason my branch didn't switch properly. 
This issue is indeed fixed, thank you!

Next issue: if you add another provider, it just disappears on restart. Just add 2 custom torrent providers and restart. The 2nd one will be missing.

**Comment by p0psicles** [comment]
Sure I'll fix in this Pr.

File: medusa/server/web/config/providers.py [hunk 1]
```diff
@@ -313,7 +313,7 @@ def saveProviders(self, newznab_string='', torrentrss_string='', provider_order=
         app.TORRENTRSS_PROVIDERS = [provider.name.upper() for provider in app.torrentRssProviderList]
 
         disabled_list = []
-        # do the enable/disable
+        # do the enable/disablef
```
Comment (duramato) [inline medusa/server/web/config/providers.py]: typo?

Comment (p0psicles) [reply medusa/server/web/config/providers.py]: Yeah

**Comment by p0psicles** [comment]
Should be all good now. Did some more extensive tests.

**Comment by p0psicles** [comment]
Anything else guys? Else i'm merging.

**Pull request merged**